core/translations/order.json

```diff
@@ -958,5 +958,6 @@
   "956": "words__title_done",
   "957": "reset__slip39_checklist_more_info_threshold",
   "958": "reset__slip39_checklist_more_info_threshold_example_template",
-  "959": "passphrase__continue_with_empty_passphrase"
+  "959": "passphrase__continue_with_empty_passphrase",
+  "960": "label_with_colon_template"
```

Comment:
This one doesn't seem to be used anywhere?

Reply:
Yep, I removed it.